### DISCUSSION TITLE
Webserver API for AX TIAX direct loading

### DISCUSSION THREAD

**Discussion by NJansDev** [siemens/simatic-s7-webserver-api#32] [Q&A] [answered]
Hello,

I want to express our appreciation for the development of the web API, which appears suitable for our use case. We are interested in using the TIAX direct loading workflow from Siemens Simatic AX and are wondering if this will have any implications or require adjustments in our use of the web API.

Thank you.

**Comment by PTKu** [comment, marked as answer]
We use this library for projects based on SIMATIC-AX, here: [AX# project](https://github.com/ix-ax/axsharp) and here: [AXOpen project](https://github.com/ix-ax/AXOpen). There's a related sub-project with this pull request: [PR#215](https://github.com/ix-ax/axsharp/pull/215). It aims to adapt the main project for the TIA portal. 

**Although we don't utilize the `TIAX` scenario, I believe it should still function in your case."**
